app/views/courses/_status_panel.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="status-box">
+  <h3 class="govuk-heading-m">Status</h3>
+  <div class="govuk-!-margin-bottom-6">
+    <%= render partial: 'content_status_tag', locals: { course: @course } %>
+  </div>
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Is it on Find?</h3>
+  <% if @course.findable? %>
+    <p class="govuk-body">
+      <%= govuk_link_to "Yes - view online", search_ui_course_page_url(provider_code: @provider.provider_code, course_code: @course.course_code) %>
+    </p>
+  <% else %>
+    <p class="govuk-body">
+      <% if @course.new_and_not_running? %>
+        No – <%= @provider.opted_in? ? 'still in draft' : 'must be published on UCAS' %>
+      <% else %>
+        No
+      <% end %>
+    </p>
+  <% end %>
+
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Applications</h3>
+  <p class="govuk-body"><%= @course.open_for_applications? ? "Open" : "Closed" %></p>
+
+  <h3 class="govuk-heading-s govuk-!-margin-0">Vacancies</h3>
+  <p class="govuk-body">
+    <%= @course.has_vacancies? ? "Yes" : "No" %>
+    <% if @provider.opted_in? %>
+      (<%= link_to 'Edit', vacancies_provider_course_path(code: @course.course_code), class: 'govuk-link' %>)
+    <% end %>
+  </p>
+
+  <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
+  <p class="govuk-body">Last published:<br><%= formatted_date(@course.changed_at) %></p>
```

Comment:
Why is `changed_at` the right way to tell when a course was last published? I think that there are some cases where `changed_at` might be updated but nothing has actually changed in terms of published data.

Reply:
What is the correct point of reference for the last time a user published this course? cc. @benilovj

Reply:
@misaka is there a more appropriatr timestamp that would reflect the publishing time? `changed_at` will incorporate new training locations being added/removed, vacancy changes etc which seem relevant to the user.

Reply:
They were my thoughts too Jake. I think this should remain and if it’s an issue at a later point then we address it separately.

Reply:
Doesn't this get updated if a draft enrichment is changed? That's the situation I was thinking of. We originally thought that this would be No Big Deal (:tm:) since it would only mean that it would be resent to UCAS but since only the published version is sent, it would be the same as the previous version.

If I'm right (need to double-check), would this be an issue with it's use here?

Reply:
@misaka oh, I wasn't aware of that, is that happening on the C# side?

Reply:
Interesting. I’ll re-look at the API and Expose the timestamp from the latest “published” enrichment instead.

Reply:
Maybe if the label were something like "Last changed" instead of "Last published", it wouldn't be misleading.

Alternatively we could take the `last_published_timestamp_utc` timestamp from the latest published enrichment and display that.

It may be worth taking that widget out of this PR into a separate one, so we don't block merging the rest.

Reply:
Done